### ISSUE TITLE
Laptop freezes when screen turns off or suspends (both s2idle and S3 sleep)

### ISSUE THREAD

**Issue by jojusuar** [pop-os/pop#3346]
```               /////////////                euclasio@pop-os 
         /////////////////////            --------------- 
      ///////*767////////////////         OS: Pop!_OS 22.04 LTS x86_64 
    //////7676767676*//////////////       Host: Dell Latitude 7280 
   /////76767//7676767//////////////      Kernel: 6.9.3-76060903-generic 
  /////767676///*76767///////////////     Uptime: 2 hours, 23 mins 
 ///////767676///76767.///7676*///////    Packages: 2017 (dpkg), 9 (flatpak), 9 
/////////767676//76767///767676////////   Shell: bash 5.1.16 
//////////76767676767////76767/////////   Resolution: 1366x768 
///////////76767676//////7676//////////   DE: GNOME 42.9 
////////////,7676,///////767///////////   WM: Mutter 
/////////////*7676///////76////////////   WM Theme: Pop 
///////////////7676////////////////////   Theme: Yaru-prussiangreen-dark [GTK2/ 
 ///////////////7676///767////////////    Icons: Yaru-prussiangreen-dark [GTK2/ 
  //////////////////////'////////////     Terminal: gnome-terminal 
   //////.7676767676767676767,//////      CPU: Intel i7-7600U (4) @ 1.950GHz 
    /////767676767676767676767/////       GPU: Intel HD Graphics 620 
      ///////////////////////////         Memory: 2742MiB / 7818MiB 
         /////////////////////
             /////////////                                                                                                   
```

**Issue/Bug Description:**
About 35% of the time I suspend the laptop (don't even know how to reproduce the problem), the screen will turn off as expected, but the LED in the power button will stay on and the laptop won't wake up, making me force reset by holding power button. BIOS managed stuff like Fn key block and the keyboard backlight remain responsive, but for example Alt+Ctrl+F1 won't do anything. Happens on both X11 and Wayland, on both s2idle and deep (S3) sleep modes. This is the journal entry for when the problem occurs, from the moment the power key is pressed to the very last line it manages to write before becoming unresponsive: 
```
Aug 07 13:03:52 pop-os systemd-logind[733]: Power key pressed.
Aug 07 13:03:52 pop-os NetworkManager[692]: <info>  [1723053832.9151] manager: sleep: sleep requested (sleeping: no  enabled: yes)
Aug 07 13:03:52 pop-os ModemManager[776]: <info>  [sleep-monitor-systemd] system is about to suspend
Aug 07 13:03:52 pop-os NetworkManager[692]: <info>  [1723053832.9153] device (p2p-dev-wlp2s0): state change: disconnected -> unmanaged (reason 'sleeping', sys-iface-state: 'managed')
Aug 07 13:03:52 pop-os NetworkManager[692]: <info>  [1723053832.9158] manager: NetworkManager state is now ASLEEP
Aug 07 13:03:52 pop-os NetworkManager[692]: <info>  [1723053832.9160] device (wlp2s0): state change: activated -> deactivating (reason 'sleeping', sys-iface-state: 'managed')
Aug 07 13:03:52 pop-os systemd[1]: Starting Network Manager Script Dispatcher Service...
Aug 07 13:03:52 pop-os systemd[1]: Started Network Manager Script Dispatcher Service.
Aug 07 13:03:52 pop-os kernel: wlp2s0: deauthenticating from 4c:42:1e:ff:5f:0d by local choice (Reason: 3=DEAUTH_LEAVING)
Aug 07 13:03:52 pop-os gnome-shell[2090]: [popx11gestures@system76.com] Extension disabled
Aug 07 13:03:52 pop-os gnome-shell[2090]: [popx11gestures@system76.com] Building entry point for GNOME Shell 42.9
Aug 07 13:03:52 pop-os gnome-shell[2090]: [popx11gestures@system76.com] No tracker, skipping
Aug 07 13:03:52 pop-os gnome-shell[2090]: [popx11gestures@system76.com] No tracker, skipping
Aug 07 13:03:52 pop-os gnome-shell[2090]: [popx11gestures@system76.com] No tracker, skipping
Aug 07 13:03:52 pop-os gnome-shell[2090]: [VERTICAL-OVERVIEW] resetting overrides
Aug 07 13:03:52 pop-os gnome-shell[2090]: [VERTICAL-OVERVIEW] disabled
Aug 07 13:03:53 pop-os wpa_supplicant[749]: wlp2s0: CTRL-EVENT-DISCONNECTED bssid=4c:42:1e:ff:5f:0d reason=3 locally_generated=1
Aug 07 13:03:53 pop-os wpa_supplicant[749]: wlp2s0: CTRL-EVENT-DSCP-POLICY clear_all
Aug 07 13:03:53 pop-os NetworkManager[692]: <info>  [1723053833.0425] device (wlp2s0): state change: deactivating -> disconnected (reason 'sleeping', sys-iface-state: 'managed')
Aug 07 13:03:53 pop-os avahi-daemon[704]: Withdrawing address record for fe80::b3bb:4662:956b:f7f2 on wlp2s0.
Aug 07 13:03:53 pop-os avahi-daemon[704]: Leaving mDNS multicast group on interface wlp2s0.IPv6 with address fe80::b3bb:4662:956b:f7f2.
Aug 07 13:03:53 pop-os avahi-daemon[704]: Interface wlp2s0.IPv6 no longer relevant for mDNS.
Aug 07 13:03:53 pop-os NetworkManager[692]: <info>  [1723053833.0499] dhcp4 (wlp2s0): canceled DHCP transaction
Aug 07 13:03:53 pop-os NetworkManager[692]: <info>  [1723053833.0500] dhcp4 (wlp2s0): activation: beginning transaction (timeout in 45 seconds)
Aug 07 13:03:53 pop-os NetworkManager[692]: <info>  [1723053833.0500] dhcp4 (wlp2s0): state changed no lease
Aug 07 13:03:53 pop-os wpa_supplicant[749]: wlp2s0: PMKSA-CACHE-REMOVED 4c:e1:75:a4:e1:4d 0
Aug 07 13:03:53 pop-os wpa_supplicant[749]: wlp2s0: PMKSA-CACHE-REMOVED c4:b3:6a:a1:e6:ed 0
Aug 07 13:03:53 pop-os wpa_supplicant[749]: wlp2s0: PMKSA-CACHE-REMOVED 84:f1:47:5e:bc:ad 0
Aug 07 13:03:53 pop-os wpa_supplicant[749]: wlp2s0: PMKSA-CACHE-REMOVED 4c:42:1e:ff:5f:0d 0
Aug 07 13:03:53 pop-os wpa_supplicant[749]: wlp2s0: PMKSA-CACHE-REMOVED 4c:42:1e:ff:5f:02 0
Aug 07 13:03:53 pop-os wpa_supplicant[749]: wlp2s0: PMKSA-CACHE-REMOVED 84:f1:47:5e:bb:0d 0
Aug 07 13:03:53 pop-os wpa_supplicant[749]: wlp2s0: PMKSA-CACHE-REMOVED 6c:d6:e3:ea:8c:4d 0
Aug 07 13:03:53 pop-os wpa_supplicant[749]: wlp2s0: PMKSA-CACHE-REMOVED 00:59:dc:8b:2a:ed 0
Aug 07 13:03:53 pop-os wpa_supplicant[749]: wlp2s0: PMKSA-CACHE-REMOVED 84:f1:47:5e:b4:2c 0
Aug 07 13:03:53 pop-os wpa_supplicant[749]: wlp2s0: PMKSA-CACHE-REMOVED 84:f1:47:9d:76:cd 0
Aug 07 13:03:53 pop-os wpa_supplicant[749]: wlp2s0: PMKSA-CACHE-REMOVED 84:f1:47:5e:bc:cd 0
Aug 07 13:03:53 pop-os wpa_supplicant[749]: wlp2s0: PMKSA-CACHE-REMOVED 84:f1:47:5e:bc:a2 0
Aug 07 13:03:53 pop-os wpa_supplicant[749]: wlp2s0: PMKSA-CACHE-REMOVED 4c:e1:75:a4:e1:42 0
Aug 07 13:03:53 pop-os wpa_supplicant[749]: wlp2s0: PMKSA-CACHE-REMOVED 00:59:dc:8b:2b:cd 0
Aug 07 13:03:53 pop-os wpa_supplicant[749]: wlp2s0: PMKSA-CACHE-REMOVED 4c:e1:75:a4:e2:6d 0
Aug 07 13:03:53 pop-os systemd-resolved[627]: wlp2s0: Bus client reset search domain list.
Aug 07 13:03:53 pop-os wpa_supplicant[749]: wlp2s0: PMKSA-CACHE-REMOVED c4:b9:cd:ff:5f:0d 0
Aug 07 13:03:53 pop-os systemd-resolved[627]: wlp2s0: Bus client set default route setting: no
Aug 07 13:03:53 pop-os avahi-daemon[704]: Withdrawing address record for 172.19.1.146 on wlp2s0.
Aug 07 13:03:53 pop-os avahi-daemon[704]: Leaving mDNS multicast group on interface wlp2s0.IPv4 with address 172.19.1.146.
Aug 07 13:03:53 pop-os avahi-daemon[704]: Interface wlp2s0.IPv4 no longer relevant for mDNS.
Aug 07 13:03:53 pop-os systemd-resolved[627]: wlp2s0: Bus client reset DNS server list.
Aug 07 13:03:53 pop-os NetworkManager[692]: <info>  [1723053833.0602] device (wlp2s0): supplicant interface state: completed -> disconnected
Aug 07 13:03:53 pop-os NetworkManager[692]: <info>  [1723053833.0610] device (wlp2s0): state change: disconnected -> unmanaged (reason 'sleeping', sys-iface-state: 'managed')
Aug 07 13:03:53 pop-os chronyd[865]: Source 185.125.190.56 offline
Aug 07 13:03:53 pop-os chronyd[865]: Source 91.189.91.157 offline
Aug 07 13:03:53 pop-os chronyd[865]: Source 3.220.42.39 offline
Aug 07 13:03:53 pop-os chronyd[865]: Source 15.237.97.214 offline
Aug 07 13:03:53 pop-os chronyd[865]: Source 52.10.183.132 offline
Aug 07 13:03:53 pop-os chronyd[865]: Source 185.125.190.57 offline
Aug 07 13:03:53 pop-os chronyd[865]: Source 18.228.202.30 offline
Aug 07 13:03:53 pop-os chronyd[865]: Source 185.125.190.58 offline
Aug 07 13:03:53 pop-os chronyd[865]: Source 179.60.247.252 offline
Aug 07 13:03:53 pop-os chronyd[865]: Source 3.134.129.152 offline
Aug 07 13:03:53 pop-os wpa_supplicant[749]: p2p-dev-wlp2s0: CTRL-EVENT-DSCP-POLICY clear_all
Aug 07 13:03:53 pop-os wpa_supplicant[749]: p2p-dev-wlp2s0: CTRL-EVENT-DSCP-POLICY clear_all
Aug 07 13:03:53 pop-os wpa_supplicant[749]: nl80211: deinit ifname=p2p-dev-wlp2s0 disabled_11b_rates=0
Aug 07 13:03:53 pop-os wpa_supplicant[749]: wlp2s0: CTRL-EVENT-DSCP-POLICY clear_all
Aug 07 13:03:53 pop-os wpa_supplicant[749]: wlp2s0: CTRL-EVENT-DSCP-POLICY clear_all
Aug 07 13:03:53 pop-os wpa_supplicant[749]: nl80211: deinit ifname=wlp2s0 disabled_11b_rates=0
Aug 07 13:03:53 pop-os gnome-shell[2090]: An active wireless connection, in infrastructure mode, involves no access point?
Aug 07 13:03:54 pop-os systemd[1]: Reached target Sleep.
Aug 07 13:03:54 pop-os systemd[1]: Starting System Suspend...
Aug 07 13:03:54 pop-os systemd-sleep[8271]: Entering sleep state 'suspend'...
Aug 07 13:03:54 pop-os kernel: PM: suspend entry (deep)
```

**Expected behavior:**
This is the journal for when the suspend process does work:

```
Aug 07 12:31:11 pop-os systemd-logind[733]: Power key pressed.
Aug 07 12:31:11 pop-os ModemManager[776]: <info>  [sleep-monitor-systemd] system is about to suspend
Aug 07 12:31:11 pop-os NetworkManager[692]: <info>  [1723051871.5519] manager: sleep: sleep requested (sleeping: no  enabled: yes)
Aug 07 12:31:11 pop-os NetworkManager[692]: <info>  [1723051871.5520] device (p2p-dev-wlp2s0): state change: disconnected -> unmanaged (reason 'sleeping', sys-iface-state: 'managed')
Aug 07 12:31:11 pop-os NetworkManager[692]: <info>  [1723051871.5526] manager: NetworkManager state is now ASLEEP
Aug 07 12:31:11 pop-os NetworkManager[692]: <info>  [1723051871.5528] device (wlp2s0): state change: activated -> deactivating (reason 'sleeping', sys-iface-state: 'managed')
Aug 07 12:31:11 pop-os systemd[1]: Starting Network Manager Script Dispatcher Service...
Aug 07 12:31:11 pop-os systemd[1]: Started Network Manager Script Dispatcher Service.
Aug 07 12:31:11 pop-os kernel: wlp2s0: deauthenticating from 00:df:1d:30:f6:cd by local choice (Reason: 3=DEAUTH_LEAVING)
Aug 07 12:31:11 pop-os wpa_supplicant[749]: wlp2s0: CTRL-EVENT-DISCONNECTED bssid=00:df:1d:30:f6:cd reason=3 locally_generated=1
Aug 07 12:31:11 pop-os wpa_supplicant[749]: wlp2s0: CTRL-EVENT-DSCP-POLICY clear_all
Aug 07 12:31:11 pop-os NetworkManager[692]: <info>  [1723051871.6890] device (wlp2s0): state change: deactivating -> disconnected (reason 'sleeping', sys-iface-state: 'managed')
Aug 07 12:31:11 pop-os avahi-daemon[704]: Withdrawing address record for fe80::b3bb:4662:956b:f7f2 on wlp2s0.
Aug 07 12:31:11 pop-os avahi-daemon[704]: Leaving mDNS multicast group on interface wlp2s0.IPv6 with address fe80::b3bb:4662:956b:f7f2.
Aug 07 12:31:11 pop-os avahi-daemon[704]: Interface wlp2s0.IPv6 no longer relevant for mDNS.
Aug 07 12:31:11 pop-os NetworkManager[692]: <info>  [1723051871.6948] dhcp4 (wlp2s0): canceled DHCP transaction
Aug 07 12:31:11 pop-os NetworkManager[692]: <info>  [1723051871.6949] dhcp4 (wlp2s0): activation: beginning transaction (timeout in 45 seconds)
Aug 07 12:31:11 pop-os NetworkManager[692]: <info>  [1723051871.6949] dhcp4 (wlp2s0): state changed no lease
Aug 07 12:31:11 pop-os wpa_supplicant[749]: wlp2s0: PMKSA-CACHE-REMOVED 00:df:1d:30:f6:c2 0
Aug 07 12:31:11 pop-os wpa_supplicant[749]: wlp2s0: PMKSA-CACHE-REMOVED dc:f7:19:3d:16:4d 0
Aug 07 12:31:11 pop-os wpa_supplicant[749]: wlp2s0: PMKSA-CACHE-REMOVED dc:f7:19:3d:16:42 0
Aug 07 12:31:11 pop-os wpa_supplicant[749]: wlp2s0: PMKSA-CACHE-REMOVED 00:df:1d:30:f6:cd 0
Aug 07 12:31:11 pop-os systemd-resolved[627]: wlp2s0: Bus client reset search domain list.
Aug 07 12:31:11 pop-os avahi-daemon[704]: Withdrawing address record for 172.20.131.209 on wlp2s0.
Aug 07 12:31:11 pop-os avahi-daemon[704]: Leaving mDNS multicast group on interface wlp2s0.IPv4 with address 172.20.131.209.
Aug 07 12:31:11 pop-os avahi-daemon[704]: Interface wlp2s0.IPv4 no longer relevant for mDNS.
Aug 07 12:31:11 pop-os systemd-resolved[627]: wlp2s0: Bus client set default route setting: no
Aug 07 12:31:11 pop-os systemd-resolved[627]: wlp2s0: Bus client reset DNS server list.
Aug 07 12:31:11 pop-os NetworkManager[692]: <info>  [1723051871.7085] device (wlp2s0): supplicant interface state: completed -> disconnected
Aug 07 12:31:11 pop-os NetworkManager[692]: <info>  [1723051871.7093] device (wlp2s0): state change: disconnected -> unmanaged (reason 'sleeping', sys-iface-state: 'managed')
Aug 07 12:31:11 pop-os gnome-shell[2090]: [popx11gestures@system76.com] Extension disabled
Aug 07 12:31:11 pop-os gnome-shell[2090]: [popx11gestures@system76.com] Building entry point for GNOME Shell 42.9
Aug 07 12:31:11 pop-os gnome-shell[2090]: [popx11gestures@system76.com] No tracker, skipping
Aug 07 12:31:11 pop-os gnome-shell[2090]: [popx11gestures@system76.com] No tracker, skipping
Aug 07 12:31:11 pop-os gnome-shell[2090]: [popx11gestures@system76.com] No tracker, skipping
Aug 07 12:31:11 pop-os chronyd[865]: Source 185.125.190.56 offline
Aug 07 12:31:11 pop-os chronyd[865]: Source 91.189.91.157 offline
Aug 07 12:31:11 pop-os chronyd[865]: Source 3.220.42.39 offline
Aug 07 12:31:11 pop-os chronyd[865]: Source 15.237.97.214 offline
Aug 07 12:31:11 pop-os chronyd[865]: Source 52.10.183.132 offline
Aug 07 12:31:11 pop-os chronyd[865]: Source 185.125.190.57 offline
Aug 07 12:31:11 pop-os chronyd[865]: Source 18.228.202.30 offline
Aug 07 12:31:11 pop-os chronyd[865]: Source 185.125.190.58 offline
Aug 07 12:31:11 pop-os chronyd[865]: Source 179.60.247.252 offline
Aug 07 12:31:11 pop-os chronyd[865]: Source 3.134.129.152 offline
Aug 07 12:31:11 pop-os gnome-shell[2090]: [VERTICAL-OVERVIEW] resetting overrides
Aug 07 12:31:11 pop-os gnome-shell[2090]: [VERTICAL-OVERVIEW] disabled
Aug 07 12:31:11 pop-os wpa_supplicant[749]: p2p-dev-wlp2s0: CTRL-EVENT-DSCP-POLICY clear_all
Aug 07 12:31:11 pop-os wpa_supplicant[749]: p2p-dev-wlp2s0: CTRL-EVENT-DSCP-POLICY clear_all
Aug 07 12:31:11 pop-os wpa_supplicant[749]: nl80211: deinit ifname=p2p-dev-wlp2s0 disabled_11b_rates=0
Aug 07 12:31:11 pop-os wpa_supplicant[749]: wlp2s0: CTRL-EVENT-DSCP-POLICY clear_all
Aug 07 12:31:11 pop-os wpa_supplicant[749]: wlp2s0: CTRL-EVENT-DSCP-POLICY clear_all
Aug 07 12:31:11 pop-os wpa_supplicant[749]: nl80211: deinit ifname=wlp2s0 disabled_11b_rates=0
Aug 07 12:31:12 pop-os gnome-shell[2090]: An active wireless connection, in infrastructure mode, involves no access point?
Aug 07 12:31:13 pop-os systemd[1]: Reached target Sleep.
Aug 07 12:31:13 pop-os systemd[1]: Starting System Suspend...
Aug 07 12:31:13 pop-os systemd-sleep[6290]: Entering sleep state 'suspend'...
Aug 07 12:31:13 pop-os kernel: PM: suspend entry (deep)
Aug 07 12:31:13 pop-os kernel: Filesystems sync: 0.026 seconds
Aug 07 12:50:58 pop-os kernel: Freezing user space processes
Aug 07 12:50:58 pop-os kernel: Freezing user space processes completed (elapsed 0.002 seconds)
Aug 07 12:50:58 pop-os kernel: OOM killer disabled.
Aug 07 12:50:58 pop-os kernel: Freezing remaining freezable tasks
Aug 07 12:50:58 pop-os kernel: Freezing remaining freezable tasks completed (elapsed 0.000 seconds)
Aug 07 12:50:58 pop-os kernel: printk: Suspending console(s) (use no_console_suspend to debug)
Aug 07 12:50:58 pop-os kernel: sd 2:0:0:0: [sda] Synchronizing SCSI cache
Aug 07 12:50:58 pop-os kernel: ata3.00: Entering standby power mode
Aug 07 12:50:58 pop-os kernel: e1000e: EEE TX LPI TIMER: 00000011
Aug 07 12:50:58 pop-os kernel: pcieport 0000:00:1c.0: Intel SPT PCH root port ACS workaround enabled
Aug 07 12:50:58 pop-os kernel: ACPI: EC: interrupt blocked
Aug 07 12:50:58 pop-os kernel: ACPI: PM: Preparing to enter system sleep state S3
Aug 07 12:50:58 pop-os kernel: ACPI: EC: event blocked
Aug 07 12:50:58 pop-os kernel: ACPI: EC: EC stopped
Aug 07 12:50:58 pop-os kernel: ACPI: PM: Saving platform NVS memory
Aug 07 12:50:58 pop-os kernel: Disabling non-boot CPUs ...
Aug 07 12:50:58 pop-os kernel: smpboot: CPU 1 is now offline
Aug 07 12:50:58 pop-os kernel: smpboot: CPU 2 is now offline
Aug 07 12:50:58 pop-os kernel: smpboot: CPU 3 is now offline
Aug 07 12:50:58 pop-os kernel: ACPI: PM: Low-level resume complete
Aug 07 12:50:58 pop-os kernel: ACPI: EC: EC started
Aug 07 12:50:58 pop-os kernel: ACPI: PM: Restoring platform NVS memory
Aug 07 12:50:58 pop-os kernel: Enabling non-boot CPUs ...
Aug 07 12:50:58 pop-os kernel: smpboot: Booting Node 0 Processor 1 APIC 0x2
Aug 07 12:50:58 pop-os kernel: CPU1 is up
Aug 07 12:50:58 pop-os kernel: smpboot: Booting Node 0 Processor 2 APIC 0x1
Aug 07 12:50:58 pop-os kernel: CPU2 is up
Aug 07 12:50:58 pop-os kernel: smpboot: Booting Node 0 Processor 3 APIC 0x3
Aug 07 12:50:58 pop-os kernel: CPU3 is up
Aug 07 12:50:58 pop-os kernel: ACPI: PM: Waking up from system sleep state S3
Aug 07 12:50:58 pop-os kernel: ACPI: EC: interrupt unblocked
Aug 07 12:50:58 pop-os kernel: pcieport 0000:00:1c.0: Intel SPT PCH root port ACS workaround enabled
Aug 07 12:50:58 pop-os kernel: pcieport 0000:00:1c.2: Intel SPT PCH root port ACS workaround enabled
Aug 07 12:50:58 pop-os kernel: pcieport 0000:00:1c.4: Intel SPT PCH root port ACS workaround enabled
Aug 07 12:50:58 pop-os kernel: DMAR: DRHD: handling fault status reg 2
Aug 07 12:50:58 pop-os kernel: DMAR: [INTR-REMAP] Request device [f0:1f.0] fault index 0x0 [fault reason 0x25] Blocked a compatibility format interrupt request
Aug 07 12:50:58 pop-os kernel: ACPI: EC: event unblocked
Aug 07 12:50:58 pop-os kernel: i915 0000:00:02.0: [drm] [ENCODER:94:DDI A/PHY A] is disabled/in DSI mode with an ungated DDI clock, gate it
Aug 07 12:50:58 pop-os kernel: i915 0000:00:02.0: [drm] [ENCODER:102:DDI B/PHY B] is disabled/in DSI mode with an ungated DDI clock, gate it
Aug 07 12:50:58 pop-os kernel: i915 0000:00:02.0: [drm] [ENCODER:118:DDI C/PHY C] is disabled/in DSI mode with an ungated DDI clock, gate it
Aug 07 12:50:58 pop-os kernel: i2c_hid_acpi i2c-DLL079F:01: i2c_hid_get_input: incomplete report (83/65369)
Aug 07 12:50:58 pop-os kernel: usb 1-5: reset high-speed USB device number 2 using xhci_hcd
Aug 07 12:50:58 pop-os kernel: ata3: SATA link up 6.0 Gbps (SStatus 133 SControl 300)
Aug 07 12:50:58 pop-os kernel: ata3.00: ACPI cmd f5/00:00:00:00:00:00(SECURITY FREEZE LOCK) filtered out
Aug 07 12:50:58 pop-os kernel: ata3.00: ACPI cmd b1/c1:00:00:00:00:00(DEVICE CONFIGURATION OVERLAY) filtered out
Aug 07 12:50:58 pop-os kernel: sd 2:0:0:0: [sda] Starting disk
Aug 07 12:50:58 pop-os kernel: ata3.00: ACPI cmd f5/00:00:00:00:00:00(SECURITY FREEZE LOCK) filtered out
Aug 07 12:50:58 pop-os kernel: ata3.00: ACPI cmd b1/c1:00:00:00:00:00(DEVICE CONFIGURATION OVERLAY) filtered out
Aug 07 12:50:58 pop-os kernel: ata3.00: configured for UDMA/133
Aug 07 12:50:58 pop-os kernel: ata3.00: Enabling discard_zeroes_data
Aug 07 12:50:58 pop-os kernel: OOM killer enabled.
Aug 07 12:50:58 pop-os kernel: Restarting tasks ... done.
Aug 07 12:50:58 pop-os kernel: random: crng reseeded on system resumption
Aug 07 12:50:58 pop-os kernel: thermal thermal_zone7: failed to read out thermal zone (-61)
Aug 07 12:50:58 pop-os systemd-resolved[627]: Clock change detected. Flushing caches.
Aug 07 12:50:58 pop-os systemd[1]: Starting Refresh fwupd metadata and update motd...
Aug 07 12:50:58 pop-os gnome-shell[2090]: JS ERROR: Failed to initialize fprintd service: Gio.DBusError: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: The name is not activatable
                                          asyncCallback@resource:///org/gnome/gjs/modules/core/overrides/Gio.js:114:23
Aug 07 12:50:58 pop-os systemd-sleep[6290]: System returned from sleep state.
Aug 07 12:50:58 pop-os kernel: PM: suspend exit
Aug 07 12:50:58 pop-os [6363]: /usr/lib/systemd/system-sleep/pop-default-settings_bluetooth-suspend failed with exit status 1.
Aug 07 12:50:58 pop-os systemd[1]: systemd-suspend.service: Deactivated successfully.
Aug 07 12:50:58 pop-os systemd[1]: Finished System Suspend.
Aug 07 12:50:58 pop-os systemd[1]: Stopped target Sleep.
Aug 07 12:50:58 pop-os systemd[1]: Reached target Suspend.
Aug 07 12:50:58 pop-os systemd-logind[733]: Operation 'sleep' finished.
Aug 07 12:50:58 pop-os systemd[1]: Stopped target Suspend.
Aug 07 12:50:58 pop-os ModemManager[776]: <info>  [sleep-monitor-systemd] system is resuming
Aug 07 12:50:58 pop-os NetworkManager[692]: <info>  [1723053058.6758] manager: sleep: wake requested (sleeping: yes  enabled: yes)
Aug 07 12:50:58 pop-os NetworkManager[692]: <info>  [1723053058.6760] device (enp0s31f6): state change: unavailable -> unmanaged (reason 'sleeping', sys-iface-state: 'managed')
Aug 07 12:50:58 pop-os kernel: e1000e 0000:00:1f.6 enp0s31f6: NIC Link is Down
Aug 07 12:50:58 pop-os NetworkManager[692]: <info>  [1723053058.7507] device (enp0s31f6): state change: unmanaged -> unavailable (reason 'managed', sys-iface-state: 'external')
Aug 07 12:50:58 pop-os systemd[1]: fwupd-refresh.service: Deactivated successfully.
Aug 07 12:50:58 pop-os systemd[1]: Finished Refresh fwupd metadata and update motd.
``` 
Goes on to continue reactivating stuff

**Comment by jojusuar** [comment]
Apparently, letting the screen turn off due to timeout can cause the same behavior. The journal seems more useful here: 

```
Aug 07 18:01:26 pop-os kernel: i915 0000:00:02.0: drm_WARN_ON((val & (1 << 30)) == 0)
Aug 07 18:01:26 pop-os kernel: WARNING: CPU: 1 PID: 2515 at drivers/gpu/drm/i915/display/intel_cdclk.c:900 skl_dpll0_update+0x152/0x170 [i915]
Aug 07 18:01:26 pop-os kernel: Modules linked in: ccm snd_seq_dummy snd_hrtimer zstd bnep snd_sof_pci_intel_skl snd_sof_intel_hda_common zram soundwire_intel snd_sof_intel_hda_mlink soundwire_cadence snd_sof_intel_hda snd_sof_pci snd_sof_xtensa_dsp snd_sof snd_hda_codec_hdmi snd_sof_utils soundwire_generic_allocation soundwire_bus snd_soc_avs snd_soc_hda_codec snd_soc_skl snd_soc_hdac_hda snd_hda_ext_core snd_soc_sst_ipc snd_ctl_led snd_soc_sst_dsp snd_soc_acpi_intel_match intel_uncore_frequency snd_soc_acpi snd_hda_codec_realtek intel_uncore_frequency_common snd_hda_codec_generic snd_soc_core snd_hda_scodec_component snd_compress ac97_bus ledtrig_audio snd_pcm_dmaengine intel_tcc_cooling x86_pkg_temp_thermal intel_powerclamp mei_pxp snd_hda_intel mei_hdcp coretemp snd_intel_dspcfg mei_wdt snd_intel_sdw_acpi kvm_intel dell_rbtn snd_hda_codec intel_rapl_msr snd_hda_core kvm dell_laptop binfmt_misc snd_hwdep iwlmvm rapl dell_smm_hwmon snd_pcm mac80211 snd_seq_midi nls_iso8859_1 libarc4 snd_seq_midi_event intel_cstate snd_rawmidi dell_wmi
Aug 07 18:01:26 pop-os kernel:  snd_seq uvcvideo iwlwifi btusb btrtl videobuf2_vmalloc snd_seq_device btintel processor_thermal_device_pci_legacy dell_smbios uvc iTCO_wdt videobuf2_memops btbcm processor_thermal_device btmtk processor_thermal_wt_hint intel_pmc_bxt dcdbas input_leds videobuf2_v4l2 wmi_bmof dell_wmi_descriptor snd_timer serio_raw cfg80211 intel_wmi_thunderbolt processor_thermal_rfim bluetooth snd videodev processor_thermal_rapl intel_rapl_common iTCO_vendor_support soundcore videobuf2_common ee1004 processor_thermal_wt_req joydev mc ecdh_generic mei_me ecc processor_thermal_power_floor processor_thermal_mbox mei intel_soc_dts_iosf intel_xhci_usb_role_switch intel_pch_thermal intel_pmc_core intel_vsec int3403_thermal pmt_telemetry int340x_thermal_zone intel_hid int3400_thermal mac_hid pmt_class sparse_keymap acpi_thermal_rel acpi_pad sch_fq_codel kyber_iosched msr parport_pc ppdev lp parport efi_pstore ip_tables x_tables autofs4 dm_crypt raid10 raid456 async_raid6_recov async_memcpy async_pq async_xor async_tx xor raid6_pq
Aug 07 18:01:26 pop-os kernel:  libcrc32c raid1 raid0 system76_io(OE) system76_acpi(OE) i915 crct10dif_pclmul hid_alps crc32_pclmul hid_generic polyval_clmulni drm_buddy polyval_generic i2c_algo_bit ghash_clmulni_intel ttm sha256_ssse3 rtsx_pci_sdmmc sha1_ssse3 rtsx_pci drm_display_helper i2c_i801 ahci e1000e libahci intel_lpss_pci i2c_smbus xhci_pci xhci_pci_renesas cec i2c_hid_acpi rc_core i2c_hid hid video intel_lpss_acpi intel_lpss wmi idma64 aesni_intel crypto_simd cryptd
Aug 07 18:01:26 pop-os kernel: CPU: 1 PID: 2515 Comm: Xwayland Tainted: G        W  OE      6.9.3-76060903-generic #202405300957~1721174657~22.04~abb7c06
Aug 07 18:01:26 pop-os kernel: Hardware name: Dell Inc. Latitude 7280/0KK5D1, BIOS 1.37.0 04/01/2024
Aug 07 18:01:26 pop-os kernel: RIP: 0010:skl_dpll0_update+0x152/0x170 [i915]
Aug 07 18:01:26 pop-os kernel: Code: 8b 7b 08 48 8b 5f 50 48 85 db 74 25 e8 b7 ee f6 f1 48 c7 c1 58 66 b6 c0 48 89 da 48 c7 c7 82 8e b9 c0 48 89 c6 e8 ee 55 49 f1 <0f> 0b e9 4a ff ff ff 48 8b 1f eb d6 48 8b 1f e9 63 ff ff ff 66 2e
Aug 07 18:01:26 pop-os kernel: RSP: 0018:ffffa16bc750b538 EFLAGS: 00010246
Aug 07 18:01:26 pop-os kernel: RAX: 0000000000000000 RBX: ffff8daa013bb650 RCX: 0000000000000000
Aug 07 18:01:26 pop-os kernel: RDX: 0000000000000000 RSI: 0000000000000000 RDI: 0000000000000000
Aug 07 18:01:26 pop-os kernel: RBP: ffffa16bc750b550 R08: 0000000000000000 R09: 0000000000000000
Aug 07 18:01:26 pop-os kernel: R10: 0000000000000000 R11: 0000000000000000 R12: ffffa16bc750b594
Aug 07 18:01:26 pop-os kernel: R13: ffff8daa1f50dc58 R14: ffff8daa1f50caf8 R15: ffff8daa1f59e028
Aug 07 18:01:26 pop-os kernel: FS:  00007c1c8e3c2d00(0000) GS:ffff8dab76480000(0000) knlGS:0000000000000000
Aug 07 18:01:26 pop-os kernel: CS:  0010 DS: 0000 ES: 0000 CR0: 0000000080050033
Aug 07 18:01:26 pop-os kernel: CR2: 0000073868f8f000 CR3: 000000011cd48005 CR4: 00000000003706f0
Aug 07 18:01:26 pop-os kernel: DR0: 0000000000000000 DR1: 0000000000000000 DR2: 0000000000000000
Aug 07 18:01:26 pop-os kernel: DR3: 0000000000000000 DR6: 00000000fffe0ff0 DR7: 0000000000000400
Aug 07 18:01:26 pop-os kernel: Call Trace:
Aug 07 18:01:26 pop-os kernel:  <TASK>
Aug 07 18:01:26 pop-os kernel:  ? show_regs+0x6c/0x80
Aug 07 18:01:26 pop-os kernel:  ? __warn+0x88/0x140
Aug 07 18:01:26 pop-os kernel:  ? skl_dpll0_update+0x152/0x170 [i915]
Aug 07 18:01:26 pop-os kernel:  ? report_bug+0x182/0x1b0
Aug 07 18:01:26 pop-os kernel:  ? handle_bug+0x46/0x90
Aug 07 18:01:26 pop-os kernel:  ? exc_invalid_op+0x18/0x80
Aug 07 18:01:26 pop-os kernel:  ? asm_exc_invalid_op+0x1b/0x20
Aug 07 18:01:26 pop-os kernel:  ? skl_dpll0_update+0x152/0x170 [i915]
Aug 07 18:01:26 pop-os kernel:  ? skl_dpll0_update+0x152/0x170 [i915]
Aug 07 18:01:26 pop-os kernel:  skl_get_cdclk+0x17/0x190 [i915]
Aug 07 18:01:26 pop-os kernel:  intel_cdclk_get_cdclk+0x15/0x30 [i915]
Aug 07 18:01:26 pop-os kernel:  gen9_disable_dc_states+0x7a/0x2d0 [i915]
Aug 07 18:01:26 pop-os kernel:  gen9_dc_off_power_well_enable+0xe/0x20 [i915]
Aug 07 18:01:26 pop-os kernel:  intel_power_well_enable+0x5c/0x80 [i915]
Aug 07 18:01:26 pop-os kernel:  intel_power_well_get+0x28/0x40 [i915]
Aug 07 18:01:26 pop-os kernel:  __intel_display_power_get_domain.part.0+0x59/0x90 [i915]
Aug 07 18:01:26 pop-os kernel:  intel_display_power_get+0x4e/0x70 [i915]
Aug 07 18:01:26 pop-os kernel:  __gt_unpark+0x27/0x80 [i915]
Aug 07 18:01:26 pop-os kernel:  __intel_wakeref_get_first+0x4b/0xc0 [i915]
Aug 07 18:01:26 pop-os kernel:  eb_select_engine+0x52b/0x640 [i915]
Aug 07 18:01:26 pop-os kernel:  i915_gem_do_execbuffer+0x482/0x1380 [i915]
Aug 07 18:01:26 pop-os kernel:  i915_gem_execbuffer2_ioctl+0x128/0x2a0 [i915]
Aug 07 18:01:26 pop-os kernel:  ? __pfx_i915_gem_execbuffer2_ioctl+0x10/0x10 [i915]
Aug 07 18:01:26 pop-os kernel:  drm_ioctl_kernel+0xb9/0x120
Aug 07 18:01:26 pop-os kernel:  drm_ioctl+0x301/0x5a0
Aug 07 18:01:26 pop-os kernel:  ? __pfx_i915_gem_execbuffer2_ioctl+0x10/0x10 [i915]
Aug 07 18:01:26 pop-os kernel:  __x64_sys_ioctl+0xa0/0xf0
Aug 07 18:01:26 pop-os kernel:  x64_sys_call+0xa68/0x24b0
Aug 07 18:01:26 pop-os kernel:  do_syscall_64+0x80/0x170
Aug 07 18:01:26 pop-os kernel:  ? __x64_sys_poll+0xc7/0x150
Aug 07 18:01:26 pop-os kernel:  ? syscall_exit_to_user_mode+0x81/0x270
Aug 07 18:01:26 pop-os kernel:  ? do_syscall_64+0x8c/0x170
Aug 07 18:01:26 pop-os kernel:  ? __x64_sys_ioctl+0xbb/0xf0
Aug 07 18:01:26 pop-os kernel:  ? syscall_exit_to_user_mode+0x81/0x270
Aug 07 18:01:26 pop-os kernel:  ? do_syscall_64+0x8c/0x170
Aug 07 18:01:26 pop-os kernel:  ? do_syscall_64+0x8c/0x170
Aug 07 18:01:26 pop-os kernel:  ? do_syscall_64+0x8c/0x170
Aug 07 18:01:26 pop-os kernel:  ? do_syscall_64+0x8c/0x170
Aug 07 18:01:26 pop-os kernel:  ? irqentry_exit+0x43/0x50
Aug 07 18:01:26 pop-os kernel:  ? sysvec_apic_timer_interrupt+0x57/0xc0
Aug 07 18:01:26 pop-os kernel:  entry_SYSCALL_64_after_hwframe+0x76/0x7e
Aug 07 18:01:26 pop-os kernel: RIP: 0033:0x7c1c8e91a94f
Aug 07 18:01:26 pop-os kernel: Code: 00 48 89 44 24 18 31 c0 48 8d 44 24 60 c7 04 24 10 00 00 00 48 89 44 24 08 48 8d 44 24 20 48 89 44 24 10 b8 10 00 00 00 0f 05 <41> 89 c0 3d 00 f0 ff ff 77 1f 48 8b 44 24 18 64 48 2b 04 25 28 00
Aug 07 18:01:26 pop-os kernel: RSP: 002b:00007ffeea228d00 EFLAGS: 00000246 ORIG_RAX: 0000000000000010
Aug 07 18:01:26 pop-os kernel: RAX: ffffffffffffffda RBX: 000000000000000b RCX: 00007c1c8e91a94f
Aug 07 18:01:26 pop-os kernel: RDX: 00007ffeea228d60 RSI: 0000000040406469 RDI: 000000000000000b
Aug 07 18:01:26 pop-os kernel: RBP: 00007ffeea228d60 R08: 000061824d94cba0 R09: 000061824cdbc1f0
Aug 07 18:01:26 pop-os kernel: R10: 000061824d81a8b8 R11: 0000000000000246 R12: 000061824cdc0fa8
Aug 07 18:01:26 pop-os kernel: R13: 000061824d81a6c0 R14: 000061824cd8ad5c R15: 00000000000000a0
Aug 07 18:01:26 pop-os kernel:  </TASK>
Aug 07 18:01:26 pop-os kernel: ---[ end trace 0000000000000000 ]---
Aug 07 18:01:26 pop-os kernel: ------------[ cut here ]------------
Aug 07 18:01:26 pop-os kernel: i915 0000:00:02.0: drm_WARN_ON(intel_cdclk_needs_modeset(&dev_priv->display.cdclk.hw, &cdclk_config))
Aug 07 18:01:26 pop-os kernel: WARNING: CPU: 1 PID: 2515 at drivers/gpu/drm/i915/display/intel_display_power_well.c:981 gen9_disable_dc_states+0x288/0x2d0 [i915]
Aug 07 18:01:26 pop-os kernel: Modules linked in: ccm snd_seq_dummy snd_hrtimer zstd bnep snd_sof_pci_intel_skl snd_sof_intel_hda_common zram soundwire_intel snd_sof_intel_hda_mlink soundwire_cadence snd_sof_intel_hda snd_sof_pci snd_sof_xtensa_dsp snd_sof snd_hda_codec_hdmi snd_sof_utils soundwire_generic_allocation soundwire_bus snd_soc_avs snd_soc_hda_codec snd_soc_skl snd_soc_hdac_hda snd_hda_ext_core snd_soc_sst_ipc snd_ctl_led snd_soc_sst_dsp snd_soc_acpi_intel_match intel_uncore_frequency snd_soc_acpi snd_hda_codec_realtek intel_uncore_frequency_common snd_hda_codec_generic snd_soc_core snd_hda_scodec_component snd_compress ac97_bus ledtrig_audio snd_pcm_dmaengine intel_tcc_cooling x86_pkg_temp_thermal intel_powerclamp mei_pxp snd_hda_intel mei_hdcp coretemp snd_intel_dspcfg mei_wdt snd_intel_sdw_acpi kvm_intel dell_rbtn snd_hda_codec intel_rapl_msr snd_hda_core kvm dell_laptop binfmt_misc snd_hwdep iwlmvm rapl dell_smm_hwmon snd_pcm mac80211 snd_seq_midi nls_iso8859_1 libarc4 snd_seq_midi_event intel_cstate snd_rawmidi dell_wmi
Aug 07 18:01:26 pop-os kernel:  snd_seq uvcvideo iwlwifi btusb btrtl videobuf2_vmalloc snd_seq_device btintel processor_thermal_device_pci_legacy dell_smbios uvc iTCO_wdt videobuf2_memops btbcm processor_thermal_device btmtk processor_thermal_wt_hint intel_pmc_bxt dcdbas input_leds videobuf2_v4l2 wmi_bmof dell_wmi_descriptor snd_timer serio_raw cfg80211 intel_wmi_thunderbolt processor_thermal_rfim bluetooth snd videodev processor_thermal_rapl intel_rapl_common iTCO_vendor_support soundcore videobuf2_common ee1004 processor_thermal_wt_req joydev mc ecdh_generic mei_me ecc processor_thermal_power_floor processor_thermal_mbox mei intel_soc_dts_iosf intel_xhci_usb_role_switch intel_pch_thermal intel_pmc_core intel_vsec int3403_thermal pmt_telemetry int340x_thermal_zone intel_hid int3400_thermal mac_hid pmt_class sparse_keymap acpi_thermal_rel acpi_pad sch_fq_codel kyber_iosched msr parport_pc ppdev lp parport efi_pstore ip_tables x_tables autofs4 dm_crypt raid10 raid456 async_raid6_recov async_memcpy async_pq async_xor async_tx xor raid6_pq
Aug 07 18:01:26 pop-os kernel:  libcrc32c raid1 raid0 system76_io(OE) system76_acpi(OE) i915 crct10dif_pclmul hid_alps crc32_pclmul hid_generic polyval_clmulni drm_buddy polyval_generic i2c_algo_bit ghash_clmulni_intel ttm sha256_ssse3 rtsx_pci_sdmmc sha1_ssse3 rtsx_pci drm_display_helper i2c_i801 ahci e1000e libahci intel_lpss_pci i2c_smbus xhci_pci xhci_pci_renesas cec i2c_hid_acpi rc_core i2c_hid hid video intel_lpss_acpi intel_lpss wmi idma64 aesni_intel crypto_simd cryptd
Aug 07 18:01:26 pop-os kernel: CPU: 1 PID: 2515 Comm: Xwayland Tainted: G        W  OE      6.9.3-76060903-generic #202405300957~1721174657~22.04~abb7c06
Aug 07 18:01:26 pop-os kernel: Hardware name: Dell Inc. Latitude 7280/0KK5D1, BIOS 1.37.0 04/01/2024
Aug 07 18:01:26 pop-os kernel: RIP: 0010:gen9_disable_dc_states+0x288/0x2d0 [i915]
Aug 07 18:01:26 pop-os kernel: Code: 4c 8b 67 50 4d 85 e4 75 03 4c 8b 27 e8 c1 15 f3 f1 48 c7 c1 98 d8 b6 c0 4c 89 e2 48 c7 c7 f0 ab b9 c0 48 89 c6 e8 f8 7c 45 f1 <0f> 0b e9 03 fe ff ff 48 8b 7b 08 45 0f b6 e8 44 0f b6 e0 4c 8b 77
Aug 07 18:01:26 pop-os kernel: RSP: 0018:ffffa16bc750b590 EFLAGS: 00010246
Aug 07 18:01:26 pop-os kernel: RAX: 0000000000000000 RBX: ffff8daa1f50c000 RCX: 0000000000000000
Aug 07 18:01:26 pop-os kernel: RDX: 0000000000000000 RSI: 0000000000000000 RDI: 0000000000000000
Aug 07 18:01:26 pop-os kernel: RBP: ffffa16bc750b5d0 R08: 0000000000000000 R09: 0000000000000000
Aug 07 18:01:26 pop-os kernel: R10: 0000000000000000 R11: 0000000000000000 R12: ffff8daa013bb650
Aug 07 18:01:26 pop-os kernel: R13: 0000000000000048 R14: ffff8daa1f50caf8 R15: ffff8daa1f59e028
Aug 07 18:01:26 pop-os kernel: FS:  00007c1c8e3c2d00(0000) GS:ffff8dab76480000(0000) knlGS:0000000000000000
Aug 07 18:01:26 pop-os kernel: CS:  0010 DS: 0000 ES: 0000 CR0: 0000000080050033
Aug 07 18:01:26 pop-os kernel: CR2: 0000073868f8f000 CR3: 000000011cd48005 CR4: 00000000003706f0
Aug 07 18:01:26 pop-os kernel: DR0: 0000000000000000 DR1: 0000000000000000 DR2: 0000000000000000
Aug 07 18:01:26 pop-os kernel: DR3: 0000000000000000 DR6: 00000000fffe0ff0 DR7: 0000000000000400
Aug 07 18:01:26 pop-os kernel: Call Trace:
Aug 07 18:01:26 pop-os kernel:  <TASK>
Aug 07 18:01:26 pop-os kernel:  ? show_regs+0x6c/0x80
Aug 07 18:01:26 pop-os kernel:  ? __warn+0x88/0x140
Aug 07 18:01:26 pop-os kernel:  ? gen9_disable_dc_states+0x288/0x2d0 [i915]
Aug 07 18:01:26 pop-os kernel:  ? report_bug+0x182/0x1b0
Aug 07 18:01:26 pop-os kernel:  ? handle_bug+0x46/0x90
Aug 07 18:01:26 pop-os kernel:  ? exc_invalid_op+0x18/0x80
Aug 07 18:01:26 pop-os kernel:  ? asm_exc_invalid_op+0x1b/0x20
Aug 07 18:01:26 pop-os kernel:  ? gen9_disable_dc_states+0x288/0x2d0 [i915]
Aug 07 18:01:26 pop-os kernel:  gen9_dc_off_power_well_enable+0xe/0x20 [i915]
Aug 07 18:01:26 pop-os kernel:  intel_power_well_enable+0x5c/0x80 [i915]
Aug 07 18:01:26 pop-os kernel:  intel_power_well_get+0x28/0x40 [i915]
Aug 07 18:01:26 pop-os kernel:  __intel_display_power_get_domain.part.0+0x59/0x90 [i915]
Aug 07 18:01:26 pop-os kernel:  intel_display_power_get+0x4e/0x70 [i915]
Aug 07 18:01:26 pop-os kernel:  __gt_unpark+0x27/0x80 [i915]
Aug 07 18:01:26 pop-os kernel:  __intel_wakeref_get_first+0x4b/0xc0 [i915]
Aug 07 18:01:26 pop-os kernel:  eb_select_engine+0x52b/0x640 [i915]
Aug 07 18:01:26 pop-os kernel:  i915_gem_do_execbuffer+0x482/0x1380 [i915]
Aug 07 18:01:26 pop-os kernel:  i915_gem_execbuffer2_ioctl+0x128/0x2a0 [i915]
Aug 07 18:01:26 pop-os kernel:  ? __pfx_i915_gem_execbuffer2_ioctl+0x10/0x10 [i915]
Aug 07 18:01:26 pop-os kernel:  drm_ioctl_kernel+0xb9/0x120
Aug 07 18:01:26 pop-os kernel:  drm_ioctl+0x301/0x5a0
Aug 07 18:01:26 pop-os kernel:  ? __pfx_i915_gem_execbuffer2_ioctl+0x10/0x10 [i915]
Aug 07 18:01:26 pop-os kernel:  __x64_sys_ioctl+0xa0/0xf0
Aug 07 18:01:26 pop-os kernel:  x64_sys_call+0xa68/0x24b0
Aug 07 18:01:26 pop-os kernel:  do_syscall_64+0x80/0x170
Aug 07 18:01:26 pop-os kernel:  ? __x64_sys_poll+0xc7/0x150
Aug 07 18:01:26 pop-os kernel:  ? syscall_exit_to_user_mode+0x81/0x270
Aug 07 18:01:26 pop-os kernel:  ? do_syscall_64+0x8c/0x170
Aug 07 18:01:26 pop-os kernel:  ? __x64_sys_ioctl+0xbb/0xf0
Aug 07 18:01:26 pop-os kernel:  ? syscall_exit_to_user_mode+0x81/0x270
Aug 07 18:01:26 pop-os kernel:  ? do_syscall_64+0x8c/0x170
Aug 07 18:01:26 pop-os kernel:  ? do_syscall_64+0x8c/0x170
Aug 07 18:01:26 pop-os kernel:  ? do_syscall_64+0x8c/0x170
Aug 07 18:01:26 pop-os kernel:  ? do_syscall_64+0x8c/0x170
Aug 07 18:01:26 pop-os kernel:  ? irqentry_exit+0x43/0x50
Aug 07 18:01:26 pop-os kernel:  ? sysvec_apic_timer_interrupt+0x57/0xc0
Aug 07 18:01:26 pop-os kernel:  entry_SYSCALL_64_after_hwframe+0x76/0x7e
Aug 07 18:01:26 pop-os kernel: RIP: 0033:0x7c1c8e91a94f
Aug 07 18:01:26 pop-os kernel: Code: 00 48 89 44 24 18 31 c0 48 8d 44 24 60 c7 04 24 10 00 00 00 48 89 44 24 08 48 8d 44 24 20 48 89 44 24 10 b8 10 00 00 00 0f 05 <41> 89 c0 3d 00 f0 ff ff 77 1f 48 8b 44 24 18 64 48 2b 04 25 28 00
Aug 07 18:01:26 pop-os kernel: RSP: 002b:00007ffeea228d00 EFLAGS: 00000246 ORIG_RAX: 0000000000000010
Aug 07 18:01:26 pop-os kernel: RAX: ffffffffffffffda RBX: 000000000000000b RCX: 00007c1c8e91a94f
Aug 07 18:01:26 pop-os kernel: RDX: 00007ffeea228d60 RSI: 0000000040406469 RDI: 000000000000000b
Aug 07 18:01:26 pop-os kernel: RBP: 00007ffeea228d60 R08: 000061824d94cba0 R09: 000061824cdbc1f0
Aug 07 18:01:26 pop-os kernel: R10: 000061824d81a8b8 R11: 0000000000000246 R12: 000061824cdc0fa8
Aug 07 18:01:26 pop-os kernel: R13: 000061824d81a6c0 R14: 000061824cd8ad5c R15: 00000000000000a0
Aug 07 18:01:26 pop-os kernel:  </TASK>
Aug 07 18:01:26 pop-os kernel: ---[ end trace 0000000000000000 ]---
```


ChatGPT says: 

> Error Location:

    The error occurs in the skl_dpll0_update and gen9_disable_dc_states functions within the i915 driver.
    These functions handle updating the Display PLL (Phase-Locked Loop) and disabling power states, respectively.

Error Message:

    The warning drm_WARN_ON((val & (1 << 30)) == 0) suggests an unexpected condition in the display clock update logic.
    gen9_disable_dc_states reports an issue with disabling power states for the display.

Module Information:

    The stack trace includes multiple calls related to power management and display clock handling in the i915 driver.

Potential Causes:

    This could be a bug in the i915 driver or an incompatibility with the specific hardware/BIOS version of your Dell Latitude 7280.
    It might also be triggered by specific power management settings or a recent kernel update.

And suggests adding the boot argument `i915.enable_dc=0` to the kernelstub. Will try it.

**Comment by jojusuar** [comment]
Apparently, `sudo kernelstub -a i915.enable_dc=0` was the solution! [Arch wiki](https://wiki.archlinux.org/title/Intel_graphics) has an entry on it, even explaining the lack of info in the journal: 

> Low-powered Intel processors and/or laptop processors have a tendency to randomly hang or crash due to the problems with the power management features found in low-power Intel chips. If such a crash happens, you will not see any logs reporting this problem. 

> i915.enable_dc=0 disables GPU power management. This does solve random hangs on certain Intel systems, notably Goldmount and Kaby Lake Refresh chips. Using this parameter does result in higher power use and shorter battery life on laptops/notebooks.
